src/app/Shared/Services/Api.service.tsx

```diff
@@ -81,19 +81,15 @@ export class ApiService {
     private readonly login: LoginService,
   ) {
     // show recording archives when recordings available
```

Comment:
This comment can probably be removed too. It used to be, long ago, that Cryostat could be deployed/run in a configuration without any archived recording storage. Nowadays we expect that there is an S3 storage provider configured, and if there is not that is just an invalid deployment configuration and not something the UI needs to be able to handle.

`this.archiveEnabled` can probably just be removed since it should always simply emit `true` now. Maybe for now to keep the patch small, the Subject can be replaced with a `BehaviorSubject` that starts with the `true` value and the call to `.next(true)` can be removed. Later we can clean up to remove it fully.